README.md

```diff
@@ -192,16 +192,20 @@ pip install /tmp/tensorflow_gpu_warpctc-1.3.0rc0-cp27-cp27mu-linux_x86_64.whl
 ### Common Voice training data
 
 The Common Voice corpus consists of voice samples that were donated through [Common Voice](https://voice.mozilla.org/).
-For automatically downloading and importing the corpus into a given directory, you can call:
+We provide an importer, that is automating the whole process of downloading and preparing the corpus.
+You just specify a target directory where all Common Voice contents should go.
+In case you already downloaded the Common Voice corpus archive, you can put it into this target directory beforehand.
```

Comment:
nit: From my perspective, it is not clear what 'put it into this target directory beforehand means`. Maybe something like:
```
In case you already downloaded the Common Voice corpus archive, you can simply run the import script on the directory where the corpus is located.
```